docs/site/tutorials/model_training_walkthrough.ipynb

```diff
@@ -836,7 +836,7 @@
     "  </td></tr>\n",
     "</table>\n",
     "\n",
-    "Swift for TensorFlow has many [optimization algorithms](https://github.com/rxwei/DeepLearning/blob/master/Sources/DeepLearning/Optimizer.swift) available for training. This model uses the SGD optimizer that implements the *[stochastic gradient descent](https://developers.google.com/machine-learning/crash-course/glossary#gradient_descent)* (SGD) algorithm. The `learningRate` sets the step size to take for each iteration down the hill. This is a *hyperparameter* that you'll commonly adjust to achieve better results."
+    "Swift for TensorFlow has many [optimization algorithms](https://github.com/rxwei/DeepLearning/blob/master/Sources/DeepLearning/Optimizer.swift) available for training. This model uses the SGD optimizer that implements the *[stochastic gradient descent](https://developers.google.com/machine-learning/crash-course/glossary#gradient_descent)* (SGD) algorithm. The `learningRate` sets the step size to take for each iteration down the hill. This is a *[hyperparameter](https://developers.google.com/machine-learning/glossary/#hyperparameter)* that you'll commonly adjust to achieve better results."
```

Comment:
Could you please update the optimizer link to https://github.com/tensorflow/swift-apis/tree/master/Sources/TensorFlow/Optimizers? The old link https://github.com/rxwei/DeepLearning is deprecated.

```suggestion
    "Swift for TensorFlow has many [optimization algorithms](https://github.com/tensorflow/swift-apis/tree/master/Sources/TensorFlow/Optimizers) available for training. This model uses the SGD optimizer that implements the *[stochastic gradient descent](https://developers.google.com/machine-learning/crash-course/glossary#gradient_descent)* (SGD) algorithm. The `learningRate` sets the step size to take for each iteration down the hill. This is a *[hyperparameter](https://developers.google.com/machine-learning/glossary/#hyperparameter)* that you'll commonly adjust to achieve better results."
```